packages/verifiable-credentials/test/create-credentials.test.ts

```diff
@@ -0,0 +1,92 @@
+// https://nodejs.org/api/assert.html
+const assert = require('assert');
+import { Utils } from '@verida/client-ts';
+import Credentials from '../src/credentials';
+import SharingCredential from '../src/sharing-credential';
+import { config, connect } from './config'
+
+
+describe('Credential tests', function () {
+    describe('Credential Units', function () {
+        this.timeout(100000);
+        let encryptedData;
+
+
+        let appContext;
+        let shareCredential;
+        let credential;
+
+        it('Login in user and create app context', async function () {
+
+            // create instances
+
+            appContext = await connect(config.PRIVATE_KEY_1);
+
+            shareCredential = new SharingCredential(appContext);
+
+            credential = new Credentials(appContext);
+
+
+            assert.equal(appContext.getContextName(), config.VERIDA_CONTEXT_NAME);
+        });
+        it('Verify Credential JWT was created correctly', async function () {
+
+            const item: any = await credential.createCredentialJWT(config.CREDENTAIL_DATA);
+
+            const testJwt = {
+                didJwtVc: item.didJwtVc
+            }
+
+            assert.deepEqual(item, testJwt, 'Expected credential didJWT was generated');
+        });
+        it('Verify a JWT signature is valid', async function () {
+
+            const item = await credential.createCredentialJWT(config.CREDENTAIL_DATA);
+
+            const data = await shareCredential.issueEncryptedCredential(item);
+
+            encryptedData = data
+
+            const jwt = await Utils.fetchVeridaUri(data.uri, appContext);
+
+            const verifiedCredential: any = await credential.verifyCredential(jwt);
+
+            assert.equal(verifiedCredential.jwt, jwt, 'JWT signature is valid');
```

Comment:
This does not test the signature is valid.

The code should look at the signature and use our encryption library (and perhaps DID client library?) to ensure the signature generated is the expected value based on who signed it.

Reply:
@taphot , i'm not sure how to do this, is there an example test for this one  ?